.dagger/test.go

```diff
@@ -279,7 +346,7 @@ func (t *Test) goTest(
 
 	return cmd.
 		WithEnvVariable("CGO_ENABLED", cgoEnabledEnv).
-		WithExec(args)
+		WithExec(args, dagger.ContainerWithExecOpts{Expect: dagger.ReturnTypeAny})
```

Comment:
Now all `test` commands will always pass? :thinking: 

We should still fail the job, probably by checking the return code.